games-action/polymc/polymc-1.4.2.ebuild

```diff
@@ -90,8 +98,11 @@ src_configure(){
 		-DCMAKE_INSTALL_PREFIX="/usr"
 		# Resulting binary is named polymc
 		-DLauncher_APP_BINARY_NAME="${PN}"
+		# Force Qt5 to avoid accidentaly building the Qt6 version and breaking things
+		-DLauncher_QT_VERSION_MAJOR=${QT_SLOT}
```

Comment:
What is the status of Qt6 support?

If it builds and works we can add a qt6 use flag to toggle between building for qt5 and qt6. Details on the use flag convention regarding handeling multiple qt versions can be found here: https://wiki.gentoo.org/wiki/Project:Qt/Policies#Handling_different_versions_of_Qt

We would have to `package.use.mask` the qt6 flag for the moment being.

Reply:
The launcher fully supports Qt6. However, it seems to me that [the quazip ebuild](https://github.com/gentoo/gentoo/blob/master/dev-libs/quazip/quazip-1.3-r1.ebuild) doesn't yet support Qt6 too, so we can't make to switch just yet I think :c

Reply:
> The launcher fully supports Qt6. However, it seems to me that [the quazip ebuild](https://github.com/gentoo/gentoo/blob/master/dev-libs/quazip/quazip-1.3-r1.ebuild) doesn't yet support Qt6 too, so we can't make to switch just yet I think :c

It does now! https://github.com/gentoo/gentoo/commit/970ed90ac451c5b90e3c2d81b639e7e2ea8466b6

For polymc you won't have to go the difficult route with `multibuild.eclass` since it is a user application and it doesn't really make sense to facilitate installing both a qt5 and qt6 version simultaneously. For quazip the `multibuild` was required because it is a library and we would otherwise get annoying blockers in the future between packages depending on `quazip[qt5]` and other packages depending on `quazip[qt6]`

Unfortunately my changes have caused conflict between your Pull Request and the main tree.

Reply:
> It does now! https://github.com/gentoo/gentoo/commit/970ed90ac451c5b90e3c2d81b639e7e2ea8466b6

Oh well xD

I'll try to make the necessary changes to add qt6 support then, I just can't be sure of when I'll be able to do it since I'm quite tied up with stuff IRL at the moment :/

Also, it seems a bit weird to add it to a bugfix release like this one. I can maybe leave 1.4.2 as it is right now, and later update the live ebuild with it, then propagate those changes to the next release?

Reply:
> Also, it seems a bit weird to add it to a bugfix release like this one. I can maybe leave 1.4.2 as it is right now, and later update the live ebuild with it, then propagate those changes to the next release?

That is also fine by me. Qt6 is still very fresh and new so there is no need to rush with adding support for it. 

Then I'll merge this as is and we can revisit qt6 in a later version bump.